File: sendsms.js
const { spawn } = require("child_process");

let regex = /[.*+?^${}()|[\]\\' ]/g;

let message = "My friend's tongue says:  I love you !!!!!+++++"

// necessay because the special characters in the message string must escaped
var escapedStr = message.replace(regex, "\\$&"); 

const command = 'sendsms';
const args = [
  '-n', '123456789',
  '-b', escapedStr
];

const sendsms = spawn(command, args);

// Event listeners for stdout and stderr
sendsms.stdout.on('data', (data) => {
  console.log(`stdout: ${data}`);
});

sendsms.stderr.on('data', (data) => {
  console.error(`stderr: ${data}`);
});

// Event listener for process exit
sendsms.on('exit', (code) => {
  console.log(`child process exited with code ${code}`);
});

File: sendsms.py
import subprocess
import re

regex = r'[.*+?^${}()|[\]\\\' ]'
message = "My friend's tongue says:  I love you !!!!!+++++"

escaped_str = re.sub(regex, r'\\\g<0>', message)

command = 'sendsms'
args = ['-n', '123456789', '-b', escaped_str]

sendsms = subprocess.Popen([command] + args, stdout=subprocess.PIPE, stderr=subprocess.PIPE)

# Event listeners for stdout and stderr
stdout, stderr = sendsms.communicate()
print(f"stdout: {stdout.decode()}")
print(f"stderr: {stderr.decode()}")

# Event listener for process exit
exit_code = sendsms.returncode
print(f"child process exited with code {exit_code}")